playground/frontend/lib/pages/embedded_playground/components/embedded_actions.dart

```diff
@@ -58,16 +59,24 @@ class EmbeddedActions extends StatelessWidget {
   void _openStandalonePlayground(PlaygroundController controller) {
     // The empty list forces the parsing of EmptyExampleLoadingDescriptor
     // and prevents the glimpse of the default catalog example.
+
+    final descriptor = controller.getLoadingDescriptor();
+    final urlDescriptor = descriptor.where((d) => d.canBePassedByUrl);
+    final jsDescriptor = descriptor.where((d) => !d.canBePassedByUrl);
+
+    final json = jsonEncode(urlDescriptor.toJson()['descriptors']);
```

Comment:
This way, if `ExamplesLoadingDescriptor` contains anything else but `.descriptors`, it is lost and we have a false sense that it is not. And it does contain `lazyLoadDescriptors` although this is not used in the embedded playground.

Also this way `['descriptors']` is compile-time unsafe. If it is renamed, we sure will forget to update it here.

I suggest working with `descriptor.descriptors` directly. This way:
1. We do not have the false sense of completeness when passing the entire `jsDescriptor` in the message.
2. We cut one level of indirection of `ExamplesLoadingDescriptor.where`.
3. We eliminate the confusion of `ExamplesLoadingDescriptor.where` losing data.

Reply:
fixed